docs/tutorials/TracingCode.adoc

```diff
@@ -0,0 +1,269 @@
+= Tutorial - Tracing a Command Execution Path
+:site-section: DeveloperGuide
+:toc: macro
```

Comment:
I noticed that the toc macro isn't used. Do you intend to have a toc? If no, can remove the toc-related attributes.

Reply:
> I noticed that the toc macro isn't used. Do you intend to have a toc? If no, can remove the toc-related attributes.